Neuro/Pathfinding/PathfindingHandler.cs

```diff
@@ -8,7 +8,7 @@ namespace Neuro.Pathfinding;
 
```

Comment:
Weird pathing will be taken for divert power to greenhouse and water plants tasks. Not a big problem since they are still accessible from the top, just saying

Resolve this comment if you don't care enough to fix it

![image](https://user-images.githubusercontent.com/32238504/230695274-c9c57f99-ebc3-431a-8f82-b9c10ab2c574.png)

Reply:
I think the issue is deeper. The spot that gets chosen is simply the closest to the task location, which means despite the task being in range on the sensible path, it wants to take the long path because that node was just closer. This can be remedied by increasing node density but then we're back at square 1.

![image](https://user-images.githubusercontent.com/72096833/230696413-269d3745-9494-4518-b78c-23d1c6a36283.png)

I'll try a few things but no guarantees.

Reply:
Upping density to `5f` seems to have resolved it. Not sure why the other path disappeared along with it but it works lol

![image](https://user-images.githubusercontent.com/72096833/230702786-c7decd62-dd76-41fc-84c4-f53c64c62b62.png)